api/turing/validation/validator.go

```diff
@@ -129,6 +129,43 @@ func checkTrafficRuleName(sl validator.StructLevel, fieldName string, value stri
 	}
 }
 
+func checkTrafficRuleCount(sl validator.StructLevel, fieldName string, rules models.TrafficRules) {
+	ruleLength := len(rules)
+	ruleCountDescription := "There should be either 0 or more than 1 traffic rules specified."
```

Comment:
I feel like this error can be more descriptive to help the users - `... more than 1 traffic rules (which must include a default rule)` specified.